compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -933,16 +934,16 @@ fn basic_type_metadata<'ll, 'tcx>(cx: &CodegenCx<'ll, 'tcx>, t: Ty<'tcx>) -> &'l
 
     // When targeting MSVC, emit MSVC style type names for compatibility with
     // .natvis visualizers (and perhaps other existing native debuggers?)
-    let msvc_like_names = cx.tcx.sess.target.is_like_msvc;
+    let cpp_like_debuginfo = cpp_like_debuginfo(cx.tcx);
```

Comment:
I wasn't sure if it would be clearer to modify comments like the above to no longer reference MSVC directly or if they should be made slightly more general with the note that this only happens for msvc so I left them alone for now. Happy to adjust them though!

Reply:
I think the comments are fine as they are for now.